index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>portfolio</title>
+</head>
+<body>
+  <header class="navigation">
+    <li class="logo">My Logo</li>
+    <li class="nav">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+      <img src="./image/Vector.svg" alt="">
+    </li>
```

Comment:
- [ ] Kindly wrap the hamburger icon in a `nav` tag as suggested by the previous reviewers. It will give screen readers an emphasis that it is being used to navigate the page, hence increasing the accessibility of your page. 😎️
You can use this structure to achieve it

```
     <nav>
         <ul class="nav">
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
            <li><img src="./image/Vector.svg" alt=""></li>
         </ul>
     </nav>
```